templates/components/streamfield/blocks/accordion.html

```diff
@@ -1,5 +1,29 @@
 {% verbatim %}
-<div class="">
-    {{ self }}
+{% load wagtailcore_tags %}
+
+<div class="container mx-auto my-16 text-black">
+    {% for accordion in accordions %}
+    <details class="max-w-3xl my-12 border-b border-grey-300">
+        {# webkit-details-marker hides the default arrow on safari #}
+        <summary
+            class="flex flex-col my-4 gap-2 hover:cursor-pointer focus-visible:outline-0 [&::-webkit-details-marker]:hidden">
```

Comment:
I'm not sure how the outline is supposed to look, but this is what I see on Firefox:

<img width="819" alt="image" src="https://github.com/user-attachments/assets/3d42703b-3a8e-4004-8bff-c33cc87b271a" />

On Chrome, it's completely invisible for some reason:

https://github.com/user-attachments/assets/a50c30f4-7a76-4e7e-ae2e-eea606419b4e

Reply:
This is how the outlining looks in Chrome for me. It's definitely showing up. So I'm not sure why it's not showing for you. 

![Screenshot 2025-03-17 at 4 33 28 PM](https://github.com/user-attachments/assets/db855b55-a972-43f0-b324-17b87ffcb44f)

I'm also not sure why your version is displaying lines beneath the content. Those shouldn't be there. Do you have a preferred method for outlines or another functional example I could look at it that you think has better practices baked in? This example originally came from the Torchbox charity kit.